cypress-io/cypress
# feat(app): Update Runs page visuals to new layout and card design

- Closes #27203 

### Additional details

- Introduce `RunsLayout` for `RunsContainer` 
  -  Git layout grouped by sha
  - w/o Git layout single group
- Update `RunCard` to
  - New breakpoint responsive design
  - Add `Debug` button that navigates to run on debug page
- Increase Latest Run count from 10 -> 100
- Fix `showDebugForCloudRun` return type, should be boolean as it's navigational and no query data is returned

![Screen Shot 2023-09-11 at 2 26 17 PM](https://github.com/cypress-io/cypress/assets/22619500/fbe45a63-580f-4011-af6e-3084e40b7614)
![Screen Shot 2023-09-11 at 2 26 07 PM](https://github.com/cypress-io/cypress/assets/22619500/7ce3a7c8-506d-41ee-9774-9902cf9cf260)

### Steps to test

- Launch Cypress with associated cloud run
- Verify latest run status matches icon

### How has the user experience changed?


### PR Tasks


- [n/a] Has a PR for user-facing changes been opened in [`cypress-documentation`](https://github.com/cypress-io/cypress-documentation)? 
- [n/a] Have API changes been updated in the [`type definitions`](https://github.com/cypress-io/cypress/blob/develop/cli/types/cypress.d.ts)?
